From 8607f7aa3258953a0eca77c878bcaee0bb91b245 Mon Sep 17 00:00:00 2001
From: William Lai <b04597@freescale.com>
Date: Tue, 24 Aug 2010 16:10:38 +0800
Subject: [PATCH] ENGR00126619 MX28: Fix compile error when CONFIG_FEC removed

Fix the compile error

Signed-off-by: William Lai <b04597@freescale.com>
(cherry picked from commit 94fb7d07d6f3f23a697cb86f4446c84e354ca51e)
---
 arch/arm/mach-mx28/mx28evk_pins.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx28/mx28evk_pins.c b/arch/arm/mach-mx28/mx28evk_pins.c
index 0490f36..0cb569f 100644
--- a/arch/arm/mach-mx28/mx28evk_pins.c
+++ b/arch/arm/mach-mx28/mx28evk_pins.c
@@ -1130,7 +1130,7 @@ int mx28evk_enet_gpio_init(void)
 void mx28evk_enet_io_lowerpower_enter(void)
 {}
 void mx28evk_enet_io_lowerpower_exit(void)
-
+{}
 #endif
 
 void __init mx28evk_init_pin_group(struct pin_desc *pins, unsigned count)
-- 
1.5.4.4

